graspy/inference/latent_distribution_test.py

```diff
@@ -93,6 +95,40 @@ class LatentDistributionTest(BaseInference):
         power or less power under which alternatives. Generally not recomended,
         as it is untested and included for experimental purposes.
 
+    align_type : str, {'sign_flips' (default), 'seedless-procrustes'} or None
```

Comment:
seems like both should be either "-" or "_", maybe "_" makes more sense in python?

Reply:
im quite i sure i took this convention from somewhere else in graspy. i think semipar? happy to change to any convention that you feel more strongly about.

Reply:
ohhhhhhh

Reply:
ur talking about underscore, not the quotations

Reply:
yes, will change to space

Reply:
(by that i mean no space)

Reply:
okay, once again didn't understand that u had unescaped underscores there. i think i had underscores everywhere anyway, except for this one docstring.

Reply:
done.